requirements.txt

```diff
@@ -1,2 +1,2 @@
-Django>=1.3
+Django>=1.3,<=1.11
```

Comment:
@feanil otherwise it picks the latest version.

Reply:
whats the issue from 1.3 onwards to 1.9